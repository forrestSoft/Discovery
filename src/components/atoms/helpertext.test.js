import React from 'react'
import {shallow, mount} from 'enzyme'
import shallowToJson from 'enzyme-to-json'

import HelperText from './helpertext'

it('render matches snapshot', () => {
  const helpertext = shallowToJson(shallow(
  	<HelperText 
      text="I am test helper text"
    />
  )) 
  expect(helpertext).toMatchSnapshot();
});
