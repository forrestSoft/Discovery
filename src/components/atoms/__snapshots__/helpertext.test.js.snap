// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render matches snapshot 1`] = `
<p>
  I am test helper text
</p>
`;
